lib/tasks/mutant.rake

```diff
@@ -0,0 +1,17 @@
+CLASSES = %w[
+  CaseRequest
+].freeze
+
+task :mutant do
+  CLASSES.each do |klass|
+    vars = 'RAILS_ENV=test NOCOVERAGE=true'
+    flags = '--use rspec --fail-fast'
+    unless system("#{vars} mutant -r ./config/environment #{flags} #{klass}")
```

Comment:
* I also recommend to always use the descendant expression in case there are no nested class / module constants, to make sure you have no false sense of coverage when future edits add them.